src/frontend/packages/core/src/shared/components/ssh-viewer/ssh-viewer.component.theme.scss

```diff
@@ -4,7 +4,7 @@
   $status-colors: map-get($app-theme, status);
   $ssh-error: map-get($status-colors, danger);
   $ssh-disconnected: mat-color($primary);
-  $ssh-background: map-get($foreground-colors, base);
+  $ssh-background: #000;
```

Comment:
understand why background in dark mode was already black and not white like the border

Reply:
Background of ssh div is black regardless (see class="xterm-viewport" style="background-color: rgb(0, 0, 0);"). We might want to consider changing this to just effect the border colour. Not important at the moment